api/src/opentrons/protocols/api_support/util.py

```diff
@@ -406,7 +409,7 @@ def _check_version_wrapper(*args, **kwargs):
                 )
             return decorated_obj(*args, **kwargs)
 
-        return _check_version_wrapper
+        return _check_version_wrapper  # type: ignore[return-value]
```

Comment:
In cases like this, I prefer using `typing.cast`, like:

```suggestion
        return cast(FuncT, _check_version_wrapper)
```

Because it feels a little more precise and careful.